recipe/conda_build_config.yaml

```diff
@@ -287,7 +287,7 @@ pin_run_as_build:
 
 # Pinning packages
 arpack:
-  - 3.5
+  - 3.6
```

Comment:
Could we go with `3.6.3` since that is the version we rebuilt? Also looks like patch releases [cause some breakage]( https://abi-laboratory.pro/index.php?view=timeline&l=arpack-ng ).

Reply:
You should either do, `3.6` and `x.x` or `3.6.3` and `x.x.x

Reply:
My understanding is that `3.6.3` with `x.x` equates to setting `3.6.3` to the lower bound, but letting the patch version float upwards (e.g. `3.6.4` would install fine with something built against `3.6.3`).

Reply:
Yes, but then you'll be stuck in 3.6.3 and you'll have to update to 3.6.4 if you want 3.6.4 in host

Reply:
That seems a bit safer given their API/ABI compatibility.